hive_metastore_client/hive_mestastore_client.py

```diff
@@ -84,6 +84,28 @@ def add_columns_to_table(
         # call alter table to add columns
         self.alter_table(dbname=db_name, tbl_name=table_name, new_tbl=table)
 
+    def drop_columns_from_table(
+        self, db_name: str, table_name: str, columns: List[str]
+    ) -> None:
+        """
+        Drops columns from a table.
```

Comment:
You could explain why we have this method (that the hive metastore doesn't have a drop command, but the alter only, and this method encapsulates this to the user)

Reply:
done!